platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/cli/utils/JtrUtils.java

```diff
@@ -81,6 +81,8 @@ public static JrsReportData scrapeTestData(
                 new VirtualTerminal().setProgressIndicatorEnabled(true).setThrowOnError(true);
         final Instant now = Instant.now();
 
+        terminal.run("gsutil");
```

Comment:
Can you add a comment saying why we are running this? Obvious now, may not be obvious later.